dokka-integration-tests/gradle/build.gradle.kts

```diff
@@ -39,3 +39,46 @@ tasks.integrationTest {
         })
     })
 }
+
+val templateProjectsDir = layout.projectDirectory.dir("projects")
+val androidSdkDir = templateProjectsDir.dir("ANDROID_SDK")
+
+tasks.withType<Test>().configureEach {
+    environment("ANDROID_HOME", androidSdkDir.asFile.invariantSeparatorsPath)
+}
+
+val updateProjectsAndroidLocalProperties by tasks.registering {
```

Comment:
`ANDROID_HOME` is required by the integration tests - possibly it could be removed.

https://github.com/Kotlin/dokka/blob/8899834c3f592747ff07b4326c5410f27a77390f/dokka-integration-tests/gradle/src/integrationTest/kotlin/org/jetbrains/dokka/it/gradle/Android0GradleIntegrationTest.kt#L26-L27

`local.properties` is useful for if you want to manually open a project (in a temp dir) that's failed a test, and you want to check it manually.